README.md

```diff
@@ -1,2 +1,30 @@
 # BubbleSort
-This project is a practice session to learn how to use Ennumerable Methods and build sorting algot
+
+This is the first project we built on Ruby. Bubble Sort counts wit two functions:
+    - bubble_sort (arr) -> for sorting numbers in incremental order
+    - bubble_sort_by (arr) -> for sorting strings by size
+
+## Objectives 
+The objectives of this project are:
+- Learn basic data types, methods, loops and syntaxis for Ruby
+- Learn the good prectices for Ruby and how to set up Gemfile
+- Build a sorting system to understand the proper use of methods, loops and printing methods
+- Learn to debug a project file    
+
+## Technologies
+- VSCode (editor)
+- Ruby (language)
+- Git/GitHub (version control)
+
+## Authors
+(@Fondem-Jr)[https://github.com/Fondem-Jr]
+(@JuliCarracedo)[https://github.com/JuliCarracedo]
+
+## How to run
+- Download/ clone this repository.
+- Install Ruby(v 2.7.2) if you haven't already.
```

Comment:
Instead of writing a whole English statement, wouldn't be better to show it?
```
$ git clone your_repo_link
$ cd folder_name
$ ruby example.rb
```
Like this.